<!-- -->
<launch>

  <node pkg="steering_controllers" type="vehicle_control.py" name="vehicle_control" output="screen">
  </node>

</launch>

